MammoNet/models/ViT.py

```diff
@@ -0,0 +1,23 @@
+from transformers import ViTForImageClassification, ViTImageProcessor
+from MammoNet.models.BaseModel import BaseModel
+import torch.nn as nn
+
+
+class VisionTransformerModel(nn.Module):
+    def __init__(self, num_classes, model_name="google/vit-base-patch16-224-in21k"):
+        super(VisionTransformerModel, self).__init__()
+        self.name = "VisionTransformerModel"
+        self.processor = ViTImageProcessor.from_pretrained(model_name)
+        self.model = ViTForImageClassification.from_pretrained(model_name, num_labels=num_classes)
+
```

Comment:
```suggestion
        # freeze backbone
        for param in self.model.vit.parameters():
            param.requires_grad = False
            
```

Reply:
vit or base model, just a suggestion since we still have to add it, since a fine-tuning a ViT is expensive and a lr of 0.0001 is too high for that anyways.